addons/purchase/purchase_view.xml

```diff
@@ -528,6 +528,55 @@
                 </form>
             </field>
         </record>
+
+        <record id="purchase_order_line_form3" model="ir.ui.view">
```

Comment:
we already have two form views of purchase order line. view id: purchase_order_line_form, purchase_order_line_form2. and now purchase_order_line_form3 ?

I do not understand why we need more than one form views ? I think, we should keep only one.